src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx

```diff
@@ -684,6 +682,8 @@ Custom groups can have either our default standard roles, or custom roles.
 
       <td>
 A basic user's abilities aren't directly defined by a specific role. A basic user can best be described as having the [**All product admin** role](#standard-roles) but without access to Full Stack Observability features ([learn more about user type](#user-type)). 
+
+When basic users are added to a group, that group's role-related restrictions apply, but a basic user can never be granted more capabilities than they start with.  
```

Comment:
Suggestion: "When basic users are added to a group, that group's role-related restrictions apply. What's more, a basic user can never be granted more capabilities than they start with."